tests/braille-specs/syc.yaml

```diff
@@ -13,7 +13,7 @@ table:
 flags: { testmode: forward }
 tests:
   # Syriac
-  - ["ܩܦ̄ܐ", "⠟⠏̄⠁"]
+  - ["ܩܦ̄ܐ", "⠟⠏⠘⠈⠤⠁"]
```

Comment:
You fixed it! This is great!

I don't know Syriac, but I'm having so much fun looking at your pull request and  learning about it. I have a question. I noticed that these Syriac letters have similar names to Hebrew letters, and they even look kind of similar to Hebrew letters. Now that I'm looking at this, it looks like the macron character is hovering over the letter Pe, just like a Rafe would hover over the Yiddish letter Pey to turn it into a Fey. So I think that might mean that this macron is really a rafe, and it might be hovering over that Pey to turn it into a Fey. In which case, wouldn't it be good to have Pe+Macron map to the same dots as a Fey in Hebrew braille (124)? And does that mean that "macron above" is like a rafe, and "macron below" is like a pasakh? And if so, are they more like letters than punctuation?

Reply:
Ok, update: I'm definitely wrong and I have no idea what I'm talking about. I read the wikipedia entry and it looks like you make a Fey in Syriac with a dot below the letter. The wikipedia entry for Syriac doesn't mention what this Macron character does, and I don't pretend to know. Great job on this!